docs/develop/rest-api/README.md

```diff
@@ -1,15 +1,27 @@
+---
+title: REST APIs
+children:
+  - course_api.md
+  - course_calculations.md
+  - resources_api.md
+  - notifications_api.md
+  - autopilot_api.md
+  - anchor_api.md
+  - plugin_api.md
+---
+
 # REST APIs
 
 REST APIs were introduced in Signal K server version 2 to provide a way for applications and plugins perform operations and ensure that the Signal K data model is consistent.
 
-The OpenAPI definitions can be found under _Documentation -> OpenAPI_ in the server Admin UI. 
+The OpenAPI definitions can be found under _Documentation -> OpenAPI_ in the server Admin UI.
 
 
 ### APIs available in Signal K server v2.0.0 and later:
 
 APIs are available via `/signalk/v2/api/<endpoint>`
 
-| API | Description | Endpoint | 
+| API | Description | Endpoint |
```

Comment:
Align formatting of Autopilot (line 28) with the previous entries.

Reply:
I'm not sure what you're suggesting. I didn't change anything else here. I think the whitespace change here was either prettier or my editor.

Reply:
Remove the ` ` from around Autopilot